eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -11,7 +11,7 @@ parameters:
   PushArgs:
   WorkingDirectory: $(System.DefaultWorkingDirectory)
   PRTitle: not-specified
-  PRBody: not-specified 
+  PRBody: ''
```

Comment:
I wouldn't have expected all these unrelated changes to come in. Bad merge?

Reply:
yes, bad merge